src/components/FileSelector/index.js

```diff
@@ -138,6 +138,20 @@ const FileSelector = React.forwardRef((props, ref) => {
                 disabled={disabled}
                 error={error}
             >
+                <RenderIf isTrue={variant === 'multiline'}>
```

Comment:
We should avoid using logic inside JSX. You can declare
`const shouldRenderCancelIcon = variant === 'multiline'`
and then 
`<RenderIf isTrue={shouldRenderCloseIcon}>`